lib/overcommit/hook/base.rb

```diff
@@ -160,22 +160,22 @@ def applicable_files
 
     private
 
-    def applicable_file?(file)
-      includes = Array(@config['include']).flatten.map do |glob|
-        Overcommit::Utils.convert_glob_to_absolute(glob)
-      end
+    def included_files
+      @included_files ||=
+        Overcommit::Utils.expand_relative_glob(@config['include'])
```

Comment:
We need to `.flatten` the array of includes/excludes since we want to support using YAML references to extend a default list of patterns. See c7471ec37961d734ca7b606be0f35c1a9ca9df51 for context. (it's my fault I didn't add tests for that functionality to catch this)